sympy/physics/optics/waves.py

```diff
@@ -10,7 +10,7 @@
 
 __all__ = ['TWave']
 
-from sympy import (sympify, pi, sin, cos, sqrt, Symbol, S,
+from sympy import (sympify, pi, sin, cos, sqrt, Number, Symbol, S,
     symbols, Derivative, atan2)
 from sympy.core.expr import Expr
 from sympy.physics.units import c
```

Comment:
Kindly change `:math:\omega is angular velocity` to `:math:\omega is angular frequency` on line 27.